plugins/woocommerce-blocks/tests/e2e/tests/attributes-filter/attribute-filter.block_theme.spec.ts

```diff
@@ -113,23 +111,21 @@ test.describe( `${ blockData.name } Block - with PHP classic template`, () => {
 			postType: 'wp_template',
 		} );
 
-		await editorUtils.enterEditMode();
+		await editor.enterEditMode();
 		await editor.insertBlock( {
 			name: 'woocommerce/filter-wrapper',
 			attributes: {
 				filterType: 'attribute-filter',
 				heading: 'Filter By Attribute',
 			},
 		} );
-		const attributeFilter = await editorUtils.getBlockByName(
-			blockData.slug
-		);
+		const attributeFilter = await editor.getBlockByName( blockData.slug );
 
 		await attributeFilter.getByText( 'Size' ).click();
 		await attributeFilter.getByText( 'Done' ).click();
 
 		await editor.saveSiteEditorEntities();
-		await page.goto( PRODUCT_CATALOG_LINK );
+		await page.goto( '/shop' );
```

Comment:
Is there a reason why we're not using the constant `SHOP_PAGE` from `@woocommerce/e2e-utils`?

Reply:
Hmm, I don't think we use that constant anywhere. 🤔 Having said that, this makes me realize that we have a conflict of aliases - our (WooCommerce Blocks) package has the same name as the [WC Core package](https://github.com/woocommerce/woocommerce/blob/ea25e2c5b2bbfa60626976fde5e5212c4038624b/packages/js/e2e-utils/README.md). Should we name our local utils `@woocommerce-blocks/e2e-utils` instead? cc @gigitux

Reply:
Good catch! Let's change it, but I'm fine with doing it on a follow-up PR if you prefer! Given that currently our project is called `@woocommerce/block-library` not sure that `@woocommerce-blocks/e2e-utils` is a good name. 

What about something like `~/e2e-utils`? This pattern is used in `@woocommerce/admin-library` project: https://github.com/woocommerce/woocommerce/blob/7b753af893a374be213d88bbfee672196200f283/plugins/woocommerce-admin/tsconfig.json

Reply:
> [...] Having said that, this makes me realize that we have a conflict of aliases - our (WooCommerce Blocks) package has the same name as the [WC Core package](https://github.com/woocommerce/woocommerce/blob/ea25e2c5b2bbfa60626976fde5e5212c4038624b/packages/js/e2e-utils/README.md).Having said that, this makes me realize that we have a conflict of aliases - our (WooCommerce Blocks) package has the same name as the [WC Core package](https://github.com/woocommerce/woocommerce/blob/ea25e2c5b2bbfa60626976fde5e5212c4038624b/packages/js/e2e-utils/README.md). [...]

Oh, I thought we were inheriting functionality from the core package and adding extra functionality to it. Are we using a complete separate package here?

Reply:
We're just [TS-aliasing our (Blocks) utilities](https://github.com/woocommerce/woocommerce/blob/fa4f3ff3f787ff5062ba50185c6fa8c364e71191/plugins/woocommerce-blocks/tsconfig.base.json#L70), which aren't related to the WC Core package. At some point in the future, though, I think having a single WooCommerce package would be the way to go. Pinging @lanej0 @adimoldovan @tammullen here for insight on potential work in this direction, i.e., the unification of the WooCommerce E2E framework, utilities, etc.

Reply:
> What about something like ~/e2e-utils? This pattern is used in @woocommerce/admin-library project: https://github.com/woocommerce/woocommerce/blob/7b753af893a374be213d88bbfee672196200f283/plugins/woocommerce-admin/tsconfig.json

On second thought, let's hold off deciding on an alias here as [we currently seem to be aliasing](https://github.com/woocommerce/woocommerce/blob/fa4f3ff3f787ff5062ba50185c6fa8c364e71191/plugins/woocommerce-blocks/tsconfig.base.json#L27-L71) everything else inside the `woocommerce-blocks` plugin folder as `@woocommerce/*`.

Reply:
> On second thought, let's hold off deciding on an alias here as [we currently seem to be aliasing](https://github.com/woocommerce/woocommerce/blob/fa4f3ff3f787ff5062ba50185c6fa8c364e71191/plugins/woocommerce-blocks/tsconfig.base.json#L27-L71) everything else inside the woocommerce-blocks plugin folder as @woocommerce/*.

Yeah, we have to update them. We didn't make any changes to this after the migration to the mono repo.